_posts/2023-02-07-Announcing-Data-Prepper-2.1.0.md

```diff
@@ -0,0 +1,253 @@
+optionally what to do when the number of events exceed the specified limit using the `when_exceeds` option. This 
+`when_exceeds` option can take either `block` or `drop` value and can either block until events are allowed or drop any 
+excess events. The `block` is useful if there is a temporary burst and the user do not want to lose any events in that 
+case. The default value for this option is `block`.
+
+And the following rate limiter configuration send 10 events per second to the open search and using `drop` option.
+
+```
+trace-normal-pipeline:
+  source:
+    pipeline:
+      name: "span-pipeline"
+  processor:
+    - aggregate:
+        identification_keys: ["serviceName"]
+        action:
+          rate_limiter:
+            events_per_second: 10
+            when_exceeds: "drop"
+        group_duration: "30s"
+        aggregate_when: "/status_code != 2 and /durationInNanos <= 1000000000"
+ sink:
+    - opensearch
+         hosts: ["https://opensearch:9200"]
+         insecure: true
+         username: "admin"
+         password: "admin"
+         index: sampled-traces
+```
+
+## OTel logs
+
+One of Data Prepper’s goals is supporting open standards. Data Prepper now supports the [OpenTelemetry](https://opentelemetry.io)
+log format. Previously, Data Prepper supported log data through the 
+[HTTP source plugin](https://opensearch.org/docs/latest/data-prepper/configuration/sources/http-source/), 
+which works well with tools such as [FluentBit](https://fluentbit.io). Some users would like to deploy the 
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) on their applications 
+and not require any other sidecars. Those users can now start to send logs to Data Prepper and then to OpenSearch.
+
+
+## OpenSearch sink improvements
+
+Running Data Prepper as an ingestion pipeline in front of OpenSearch is an important capability for Data Prepper. This 
+release now adds hew enhancements to Data Prepper’s OpenSearch sink to make it even more useful with OpenSearch.
+
+Data Prepper now supports routing traces and logs to different indices dynamically. Pipeline authors can define an 
+index name using a format string which can include properties from different events. With this capability, Data Prepper 
+can support an arbitrary number of indices using a single sink. For example, if you have logs with different 
+application identifiers, you could route events to an index specifically for each application.
+
+Data Prepper also now allows routing documents to specific OpenSearch shards by use of the existing `routing` parameter 
+in OpenSearch. The new `routing_field` property on the OpenSearch sink will use properties from events to specify how to 
+route within OpenSearch. Most users will prefer to allow OpenSearch to generate document ids and choose the shard 
+from that. But, in some cases users need to specify these values and now Data Prepper offers users that option.
```

Comment:
What exactly is meant by "choose the shard from that"? Could we add a noun here or explain a bit more?
```suggestion
from that. However, Data Prepper now allows you to specify these values explicitly.
```